aviary/variable_info/variable_meta_data.py

```diff
@@ -2302,6 +2316,20 @@
     desc='specifies engine type used for engine mass calculation',
 )
 
+add_meta_data(
+    Aircraft.Engine.USE_PROPELLER_MAP,
+    meta_data=_MetaData,
+    historical_name={"GASP": None,
+                     "FLOPS": None,
+                     "LEAPS1": None
+                     },
+    option=False,
+    default_value=False,
+    types=bool,
+    units="unitless",
```

Comment:
Since this is a bool, and is a modeling decision, I would set `options` to True.

Reply:
I meant to set `True` but somehow made this mistake. Changed.